www/_posts/2023-07-10-cordova-ios-7.0.0.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+author:
+    name: Bryan Ellis
+title:  "Cordova iOS 7.0.0 Released!"
+categories: announcements
+tags: news releases
+---
+
+We are happy to announce that we have just released `Cordova iOS 7.0.0`! This is one of Cordova's supported platforms for building iOS applications.
+
+* [cordova-ios@7.0.0](https://www.npmjs.com/package/cordova-ios)
+
+## Release Highlights
+
+**To upgrade:**
+
+```bash
+cordova platform remove ios
+cordova platform add ios@7.0.0
+```
+
+**To install:**
+
+```bash
+cordova platform add ios@7.0.0
+```
+
+**BREAKING CHANGES:**
+
+* **Removal of `podspec` type from `framework` tag**
+
+    Since Cordova-iOS 5.0.0, the new `podspec` tag was added to improve the readability and support of features that CocoaPods provided.
+
+    This release officially removes the old `type="podspec"` implementation of the `framework` tag. If you maintain plugins and still using the `framework` tag to load pod specs, it is recommended to migrate to the newer implementation.
+
+    For implementation specifications, please see our [Apache Cordova - Podspec](https://cordova.apache.org/docs/en/dev/plugin_ref/spec.html#podspec-) docs.
+
+* **Removed default `CONFIGURATION_BUILD_DIR` overrides**
+
+* **Node Support**
+
+    We have dropped support for Node 14.x and increase the minimum Node requirement to greater than or equal to 16.13.0.
+
+* **Dropped Platform Binraries**
+
+    We no longer supply or package platform-centric workflow binraries in the `cordova-ios` npm package or GitHub repository.
+
+* **Rename `Images.xcassets` to `Assets.xcassets`**
+
+* **Remove deprecated API `colorFromColorString` from `CDVViewController`**
+* **Privatized the `CDVCommandDelegateImpl` class.**
+
+**New Features:**
+
+* **Added `LimitsNavigationsToAppBoundDomains` preference configuration**
+
+    This preference allows you to use cookie authentication or browser APIs but requires the value to be set to `YES`. The default value is `NO`.
+
+* **Support Apple Cloud Distribution Signing**
+
+* **Enable Mac Catalyst Support**
```

Comment:
We probably want to say something like this:

> Apps built with cordova-iOS can now be made to target macOS using the Catalyst runtime. You can enable this by checking the "macOS - Catalyst" checkbox in Xcode project settings, and then building for a macOS target in Xcode.
>
> Currently, building for Catalyst from the command-line `cordova` tool is not supported.

Reply:
Resolved in: https://github.com/apache/cordova-docs/pull/1321/commits/405375f5415928328f2d35042ba5e93e441a9293